release-notes/opensearch.release-notes-3.3.0.md

```diff
@@ -32,7 +32,6 @@ Compatible with OpenSearch and OpenSearch Dashboards version 3.3.0
 * Add all-active ingestion as docrep equivalent in pull-based ingestion ([#19316](https://github.com/opensearch-project/OpenSearch/pull/19316))
 * Adding logic for histogram aggregation using skiplist ([#19130](https://github.com/opensearch-project/OpenSearch/pull/19130))
 * Add skip_list param for date, scaled float and token count fields ([#19142](https://github.com/opensearch-project/OpenSearch/pull/19142))
-* Enable skip_list for @timestamp field or index sort field by default ([#19480](https://github.com/opensearch-project/OpenSearch/pull/19480))
```

Comment:
Lets keep the change in the release notes since 3.3.0 is shipped with this change @asimmahmood1